libs/core/src/lib/form/form-control/form-control.component.scss

```diff
@@ -1,3 +1,3 @@
 @import '~fundamental-styles/dist/input';
 @import '~fundamental-styles/dist/textarea';
-@import '~fundamental-styles/dist/form-select';
+// @import '~fundamental-styles/dist/form-select';
```

Comment:
can we remove it?

Reply:
no, it can't be removed as it will break the build. There's another PR that needs to be merged (brings the Breaking Changes from 0.13.0 to ngx) and then this PR will be rebased to master and then the imports will be removed. Please check the Note in bold in the PR description.